tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -430,6 +430,23 @@ def config_bridge_domain(agent, test_bd, stepinfo='bridge-domain config:')
   end
 end
 
+# Helper for setting system bridge-domain configs
```

Comment:
Nexus folks - remind me, why are we writing feature-specific methods in utilitylib?